posthog/temporal/tests/batch_exports/test_bigquery_batch_export_workflow.py

```diff
@@ -209,6 +280,7 @@ async def test_insert_into_bigquery_activity_inserts_data_into_bigquery_table(
         count_other_team=0,
         properties=None,
         person_properties=None,
+        event_name="test-no-prop-{i}",
```

Comment:
Required for sorting in `assert_clickhouse_records_in_bigquery` to work. Otherwise there will be valid events with duplicate `event_names` that will not compare equals.

Reply:
The alternative was to do a higher dimension sort key (i.e. `(event, timestamp)`), but it's hard to make that work with custom schemas.